src/TestInternalGrainInterfaces/TestInternalGrainInterfaces.csproj

```diff
@@ -39,6 +39,14 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="ActivationGCTestGrainInterfaces.cs" />
```

Comment:
Looks like most of these interfaces aren't related to the test we are migrating. They are used by other tests. So let's add them in a PR with those tests.

Also, looks like most if not all of them don't use internal types and should be moved to TestGrainInterfaces instead.